onnxruntime/core/providers/shared_library/provider_bridge_provider.cc

```diff
@@ -9,7 +9,6 @@
 #include "core/providers/shared/common.h"
 
 #include "core/common/inlined_containers.h"
-#include "core/framework/murmurhash3.h"
```

Comment:
Should not remove because this is now used by CANN: https://github.com/microsoft/onnxruntime/pull/13728